page-for-post-type.php

```diff
@@ -259,6 +259,8 @@ public function update_post_type( $post_type, $args ) {
 				}
 			}
 
+			do_action("page_for_{$post_type}_rewrite_rules", $archive_slug, $post_type, $args);
```

Comment:
Need to match the existing code style here, it's also good to add doc blocks to actions and filters.
```suggestion
			/**
			 * Action called after generating initial rewrite rules for a page as post type archive.
			 *
			 * @param string $archive_slug The slug for the post type archive page.
			 * @param string $post_type The post type name.
			 * @param WP_Post_Type $post_type_object The post type object.
			 */
			do_action( "page_for_{$post_type}_rewrite_rules", $archive_slug, $post_type, $args );
```